Merge END_OF_TRACK with END_OF_TRACK_AND_GROUP (#755)
moq-wg/moq-transport: draft-ietf-moq-transport.md

         in this Group. This SHOULD be cached. A publisher MAY use an end of
         Group object to signal the end of all open Subgroups in a Group.

* 0x4 := Indicates end of Track. GroupID is either the largest group produced
         in this track and the ObjectID is one greater than the largest object
         produced in that group, or GroupID is one greater than the largest
         group produced in this track and the ObjectID is zero. This status

>          implies the end of the specified group. An object with this status

Reviewer: ```suggestion
         also indicates the last group has ended. An object with this status
```

         in this Group. This SHOULD be cached. A publisher MAY use an end of
         Group object to signal the end of all open Subgroups in a Group.

* 0x4 := Indicates end of Track. GroupID is either the largest group produced
         in this track and the ObjectID is one greater than the largest object
         produced in that group, or GroupID is one greater than the largest
         group produced in this track and the ObjectID is zero. This status

>          implies the end of the specified group. An object with this status

Reviewer 2: s/This status/ The latter status/

Author: There's only one status now, so I think "This status" is correct.

         that has a Group ID less than any other GroupID, or an ObjectID less
         than or equal to the largest in the specified group, is a protocol

>          error, and the receiver MUST terminate the session. This SHOULD be

Reviewer: Should we say something about how this shouldn't be propagated either?  Otherwise a publisher might be able to close every downstream subscriber's session.

Author: I guess a note to that effect might be helpful.

Reviewer: I filed #775 as a general issue.  It's possible we should say that once in the doc and then we don't need to call it out for each individual message?

Author: #806 addresses this
